lib/explorer/series.ex

```diff
@@ -26,6 +26,8 @@ defmodule Explorer.Series do
     * `{:f, size}` - a 64-bit or 32-bit floating point number
     * `{:s, size}` - a 8-bit or 16-bit or 32-bit or 64-bit signed integer number.
     * `{:u, size}` - a 8-bit or 16-bit or 32-bit or 64-bit unsigned integer number.
+    * `{:decimal, precision, scale}` - a 128-bit signed integer number representing a decimal,
+      with a scale and precision. This unwraps to `Decimal`, using the `:decimal` package.
```

Comment:
What does it mean to have nil precision or nil scale?

Reply:
This is not clear reading Polars' code base. But seems that:
 - precision is ignored most of the time. The only case when I see it appearing is when casting a lazy series to a decimal without a given precision (nil) - it uses the maximum value of 38. But if it's present, it will remove all numbers that are above that precision. If this is `nil`, then the maximum precision is used.
 - scale is optional, but always defaults to zero. So I think the correct default value could be zero.

Reply:
It makes sense. Since it is decimal128, the precision is always 38, then it is a matter of deciding of where to put the dot (the scale). So I think we can default everywhere to 38 instead of `nil`. WDYT?

Reply:
OK, makes sense! I will change that. I will do the same for the `inter_type/1` function as well, OK? https://github.com/elixir-explorer/explorer/pull/988#discussion_r1795999861

Reply:
Yes, let's try defaulting to 38 everywhere instead of computing it.